### DISCUSSION TITLE
Entering API key

### DISCUSSION THREAD

**Discussion by MikeEgilsson** [TransformerOptimus/SuperAGI#675] [Q&A]
Really basic newbie question - when entering API keys, are the first letters and hyphen required, or just the alphanumeric sequence ?
Examples:
OpenAI secret API keys start with sk-xxxxxxxx....._
Organization ID starts with org-xxxxxxxx.....

GitHub Personal access tokens start with ghp_xxxxxx.........

Admittedly a simple question, but I'm really interested to get SuperAGI working....

**Comment by neelayan7** [comment]
Yup, all the letters are required.

**Reply by MikeEgilsson** [reply]
Thanks, much obliged
